test.py

```diff
@@ -1,12 +1,30 @@
 from crawler.rss import RSSCrawler
 from crawler.github import GithubCrawler
-from crawler.config import RSSConfig
-import json
+from crawler.config import RSSConfig as conf
+import json, sys
 
-# RSS test
-rss_result = RSSCrawler("The Verge")
-print(json.dumps(rss_result, indent=4))
+def test_rss():
```

Comment:
This is ok for now, but: if this is meant to be a unit test, there should be unit testing framework. Please add a TODO to add that framework, and use mock data to test the crawl method rather than really running the Soup crawler. We should be testing extract() method, by passing in mock BeautifulSoup object.

Reply:
Now `unittest` module and some test codes are available on #2.